src/Powercord/plugins/pc-moduleManager/util/cloneRepo.js

```diff
@@ -13,12 +14,21 @@ module.exports = async function download (url, powercord, type) {
       dir = join(__dirname, '..', '..', '..', 'themes');
       break;
   }
-  console.log(dir);
-  const repoName = url.match(/([\w-]+)\/?$/)[1];
+
+  const urlMatch = url.match(REPO_URL_REGEX);
+  if (!urlMatch) {
+    console.error('Could not parse URL');
```

Comment:
should probably log the actual url here, otherwise this log is pretty meaningless